unittests/mocks/aal/camera_compatibility_layer_capabilities.h

```diff
@@ -29,6 +29,7 @@ typedef enum
 {
     FLASH_MODE_OFF,
     FLASH_MODE_AUTO,
+    FLASH_MODE_RED_EYE,
```

Comment:
Don't put enums in the middle, put them in the end, this way we keep abi somewhat.

Reply:
To be honest, I am not sure what you try to tell me or want me to change or why. Should I add the new value at the very last position? Why would that make a difference? Should the order be changed in all places?
Does c++ handle things different when the order is different? Is that value different to the others? :thinking:

Reply:
Yes, its a good habit to add it at the last position. Why? Because you never know there might exist a copy of the enum definition, and when you insert you break existing values (in other definitions) whereas appending the value just breaks the new value ;) - Even if here we might know its the only definition.

Reply:
Ah, thanks for the explanation. I will change that then.